stdlib/public/core/Integers.swift.gyb

```diff
@@ -237,11 +255,14 @@ extension BinaryInteger {
     fatalError()
   }
 
+  @_transparent
   public func signum() -> Self {
-    // FIXME(integers): implement
-    fatalError()
+    if self < (0 as Self) { return -1 }
```

Comment:
if these `as Self`s are really needed I think there’s a compiler bug that should be reported.  Otherwise we should just say `0`.